setup.py

```diff
@@ -19,8 +19,8 @@
 setup(
   name = 'graphene_file_upload',
   packages = ['graphene_file_upload'], # this must be the same as the name above
-  version = '0.1.2',
-  description = 'Lib for adding file upload functionality to GraphQL mutations in Graphene Django',
+  version = '1.0.0',
```

Comment:
I'm not sure this library is at `1.0` yet, since it's still not very well tested. Maybe start with `1.0.0-alpha.1` or `1.0.0-rc.1`? That way, it's clear that there may be changes coming if any tests are an issue